qcodes/utils/metadata.py

```diff
@@ -55,8 +58,8 @@ def snapshot(self, update=False):
 
         return snap
 
-    def snapshot_base(self, update: bool=False,
-                      params_to_skip_update: Sequence[str]=None):
+    def snapshot_base(self, update: bool = False,
```

Comment:
it's a bit weird that already Metadatable knows about parameters (due to `params_to_skip_update`) while the parameters appear only in the instrument.... but that's not to be dealt with in this PR, of course, just nagging.